app/new/template.hbs

```diff
@@ -0,0 +1,23 @@
+{{outlet}}
+
+<header class="new-run">
+  <h1 class="new-run__header">New Run</h1>
+</header>
+
+<form class="form" {{action saveNewRun on="submit"}}>
+  <div class="column">
+    <div class="form-fields">
+      {{input class="form-fields__input" type="text" placeholder="Run Time" value=runTime}}
+    </div>
+    <div class="form-fields">
+      {{input class="form-fields__input" type="text" placeholder="Run Date" value=runDate}}
+    </div>
+    <div class="form-fields">
+      {{input class="form-fields__input" type="text" placeholder="Race Notes" value=raceNote}}
+    </div>
+  </div>
+  <div class="buttons">
+    {{#link-to "index"}}<button class="buttons__button cancel">Cancel</button>{{/link-to}}
```

Comment:
Add `type="button"` to this button so that the cancel isn't run when I hit ENTER